jupyterlab_server/config.py

```diff
@@ -242,3 +249,24 @@ def _default_translations_api_url(self):
     @default('tree_url')
     def _default_tree_url(self):
         return ujoin(self.app_url, 'tree/')
+
+
+def _get_config_dir(user=False, sys_prefix=False):
+    """Get the location of config files for the current context
+    Returns the string to the environment
+    Parameters
+    ----------
+    user : bool [default: False]
+        Get the user's .jupyter config directory
+    sys_prefix : bool [default: False]
+        Get sys.prefix, i.e. ~/.envs/my-env/etc/jupyter
+    """
+    if user and sys_prefix:
+        sys_prefix = False
```

Comment:
if user is true, sys_prefix won't ever get looked at below. Why set sys_prefix to False?

Reply:
Would it be more elegant to just have a level parameter that can be 'user', 'sys_prefix', or 'system'?